buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts

```diff
@@ -122,6 +122,7 @@ tasks.withType<Test>().all {
     val jetbrainsCoreTestResources = project(":plugin-toolkit:jetbrains-core").projectDir.resolve("tst-resources")
     systemProperty("idea.log.config.properties.file", jetbrainsCoreTestResources.resolve("toolkit-test-log.properties"))
     systemProperty("org.gradle.project.ideProfileName", ideProfile.name)
+    systemProperty("aws.dev.useDAG", true)
```

Comment:
Should this be initialized as false now? Would it cause a conflict over the registry option?

Reply:
Discussed offline